examples/nextjs-spa/README.md

```diff
@@ -0,0 +1,14 @@
+```bash
+npm run dev
+```
+
+[http://localhost:3000](http://localhost:3000)
+
+## Proxy
+
+Instead of using the built in Ory proxy, we are using Ory CLI's tunnel with a
+default local variable:
```

Comment:
```suggestion
## Ory Tunnel

This example uses the Ory tunnel to mirror Ory API's on your local machine for cookies to function properly and to prevent CORS errors. 

This environment variable is used by the Ory SDK and in this case contains the localhost URL of our Ory CLI tunnel.

```